src/main/cpp/utils/Logger.h

```diff
@@ -100,72 +115,66 @@ class Logger
         void LogError
         (
             LOGGER_LEVEL            level,
-            const std::string&      locationIdentifier,     
-            const std::string&      message                 
+            const std::string&      locationIdentifier,
+            const std::string&      message
         );
 
-
-        /// @brief Write a message to the dashboard
-        /// @param [in] std::string: classname or object identifier
-        /// @param [in] std::string: message
-        void OnDash
-        (
-            const std::string&   locationIdentifier,    
-            const std::string&   message  
-        );              
-
-        /// @brief Write a message to the dashboard
+        /// @brief Write a message and value to the dashboard, for monitoring the value
         /// @param [in] std::string: classname or object identifier
-        /// @param [in] bool: boolean that should be written
+        /// @param [in] double: value that should be written (int or bool could also be 'doubled' and sent)

```

Comment:
why did you remove the string version and the bool version and replace with a double version?